Sources/Kafka/Configuration/KafkaConfiguration+Security.swift

```diff
@@ -122,104 +118,123 @@ extension KafkaConfiguration {
             }
         }
 
-        internal enum _TLSConfiguration: Sendable, Hashable {
-            case keyPair(
+        /// Configuration for the TLS verification of the client.
+        public struct ClientIdentity: Sendable, Hashable {
+            internal enum _ClientIdentity: Sendable, Hashable {
+                case keyPair(
+                    privateKey: PrivateKey,
+                    certificate: LeafAndIntermediates
```

Comment:
Leaf and optional intermediates can be more than one
```suggestion
                    certificates: LeafAndIntermediates
```